### PR TITLE
Targeted sequencing fixes

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/kit/KitDescriptor.java [hunk 1]
```diff
@@ -81,13 +82,33 @@
...
+
+  public void setChangelog(Collection<ChangeLog> changelog) {
+    this.changelog = changelog;
+  }
+
+  public void setTargetedSequencing(Set<TargetedSequencing> targetedSequencing) {
+    this.targetedSequencing = targetedSequencing;
+  }
+
+  public long getKitDescriptorId() {
```
Comment (djcooke) [inline core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/kit/KitDescriptor.java]: There are already `getId()` and `setId(Long)` methods in this class

**Comment by tonydebat** [comment]
When I use a single library form to generate a library dilution the number of available targeted sequencing values is restricted by the library kit:
image
When I propagate the same library into a library dilution I'm shown a large list of targeted sequencing values that are not restricted by the library kit. Also, the list contains duplicates.
image

**Comment by tonydebat** [comment]
Uggg... and then I cleared my cache. 👍

File: core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/kit/KitDescriptor.java (hunk 1 above)
Comment (apmasell) [inline core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/kit/KitDescriptor.java]: Sorry. Over eager autogeneration.

**Pull request merged**